demos/homeassistant DSO schedules/demo_instructions.md

```diff
@@ -0,0 +1,684 @@
+# Home Assistant with DSO profiles
+These demo instructions describe a proof-of-concept set-up to get capacity profiles communicated by a DSO (Distribution System Operator, such as Liander) available within Home Assistant. After following these instructions you should be able to reproduce the video below.
+
+https://github.com/alliander-opensource/ReLevENT/assets/58041016/8ab08547-4a0c-4b70-a9d6-2518d192b12d
+
+Please note these are *demo* instructions, as of February 2024 this solution is not in production.
```

Comment:
You can add: The documentation can be used as startingpoint for further development and/or to explore the concept.